cli/lib/kontena_cli.rb

```diff
@@ -158,11 +159,18 @@ def self.logger
       require 'kontena/cli/log_formatters/strip_color'
       logger.formatter = Kontena::Cli::LogFormatter::StripColor.new
     end
-    logger.level = ENV["DEBUG"] ? Logger::DEBUG : Logger::INFO
+    logger.level = debug? ? Logger::DEBUG : Logger::INFO
     logger.progname = 'CLI'
     @logger = logger
   end
 
+  DEBUG_KEYWORDS = Set.new(%w(true api websocket plugin))
+
+  def self.debug?
+    DEBUG_KEYWORDS.include?(ENV['DEBUG'].to_s)
```

Comment:
Hmm... so something like `DEBUG=1 kontena ...` would have enabled debug output before, but now gets ignored... probably not worth worrying about this too much, although maybe consider special-casing `false` instead?

Reply:
heh, true. `== false` would've been so much simpler.